### PR TITLE
Server certificate for API 800,1000,1200

### PR DESCRIPTION
**Pull request by nabhajit-ray**
Server certificate for API 800,1000,1200

### CODE REVIEW

File: oneview/resource_server_certificate.go [hunk 1]
```diff
@@ -0,0 +1,513 @@
...
+		Category:    d.Get("category").(string),
+		Created:     d.Get("created").(string),
+		Description: utils.NewNstring(d.Get("description").(string)),
+		ETAG:        d.Get("etag").(string),
+		Modified:    d.Get("modified").(string),
+		Name:        d.Get("name").(string),
+		State:       d.Get("state").(string),
+		Status:      d.Get("status").(string),
+		Type:        d.Get("type").(string),
+		URI:         utils.NewNstring(d.Get("uri").(string)),
```
Comment (AnushaY1916) [inline oneview/resource_server_certificate.go:280]: URI will be created post creation of this resource, so reading this attribute is not needed.

File: docs/d/server_certificate.html.markdown [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+output "oneview_server_certificate_value" {
+ value = "${data.oneview_server_certificate.test.uri}"
+}
+```
+
+## Argument Reference
+
+* `remote_ip` - (Required) Either alias_name or remote_ip  needs to be provided. Remote ip is used to fetch the server certificate of a remote host.
```
Comment (AnushaY1916) [inline docs/d/server_certificate.html.markdown:27]: Making both attributes required is not correct. Describe correctly that any one of these attributes can be used to get the Server certificate.

File: docs/d/server_certificate.html.markdown [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+# oneview\_server\_certificate
+
+Use this data source to access the attributes of a server certificate.
+
+## Example Usage
+
+```hcl
+data "oneview_server_certificate" "test" {
+ name = "Test server certificate"
```
Comment (AnushaY1916) [inline docs/d/server_certificate.html.markdown:17]: It should be either remote IP or alias name, here its just name which is incorrect.

File: examples/server_certificate.tf [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+	ov_apiversion = <ov-apiversion>
+	ov_ifmatch = "*"
+}
+
+data "oneview_scope" "scope_obj" {
+        name = "test_scope"
+}
+
+resource "oneview_server_certificate" "ServerCertificate" {
+
```
Comment (AnushaY1916) [inline examples/server_certificate.tf:14]: Format the file properly without many empty lines and spaces.

File: examples/server_certificate.tf [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+                                type="CertificateDetailV2"
+	                        alias_name = "TestServerCertificate"}
+]
+}
+*/
+/* Testing data source
+
+data "oneview_server_certificate" "sc" {
+/* One of the date either alias_name or remote_ip can used for data source
+         alias_name = "hm_cert"
```
Comment (AnushaY1916) [inline examples/server_certificate.tf:37]: Comment saying that any one of these fields can be used to get the data source.

File: examples/server_certificate.tf (hunk 4 above)
Comment (nabhajit-ray) [reply examples/server_certificate.tf]: Done

File: examples/server_certificate.tf (hunk 5 above)
Comment (nabhajit-ray) [reply examples/server_certificate.tf:36]: Done

File: oneview/resource_server_certificate.go (hunk 1 above)
Comment (nabhajit-ray) [reply oneview/resource_server_certificate.go]: Done

File: docs/d/server_certificate.html.markdown (hunk 3 above)
Comment (nabhajit-ray) [reply docs/d/server_certificate.html.markdown]: Done

File: docs/d/server_certificate.html.markdown (hunk 2 above)
Comment (nabhajit-ray) [reply docs/d/server_certificate.html.markdown]: Done

**Pull request merged**